# Álgebra para analizar objetos geométricos

# Clase 4

## Paralelismo y perpendicularidad

Pensemos que tenemos dos rectas:

* Primera recta, $y=m_1x+b_1$
* Segunda recta, $y=m_2x+b_2$

Estas rectas son paralelas si

$$ m_1=m_2 $$

Estas rectas son perpendiculares si

$$ m_1m_2=-1 $$

* ***Ejemplo*** Tenemos la ecuación de la recta $y=6x-2$. Construye otra recta que sea paralela.

Si quiero otra recta que sea paralela, entonces la ecuación de esta recta tiene que tener pendiente $m=6$.

$$ y=6x+b $$

Puedo tomar cualquier $b$ y seguirá siendo paralela a la ecuación del ejemplo.

* ***Ejemplo*** Tenemos la ecuación de la recta $y=\dfrac{3}{2}x-2$. Construye otra recta que sea perpendicular.

Si quiero otra recta que sea perpendicular, entonces la ecuación de esta recta tiene que tener pendiente $m=-\dfrac{2}{3}$.

$$ y=-\dfrac{2}{3}x+b $$

Puedo tomar cualquier $b$ y seguirá siendo paralela a la ecuación del ejemplo.

*(Básicamente, si queremos la pendiente perpendicular basta con que volteemos la fracción de nuestra pendiente y su signo.)*

* ***Ejemplo*** Tenemos la ecuación de la recta $y=6x-2$. Construye otra recta que sea perpendicular.

Si quiero otra recta que sea perpendicular, entonces la ecuación de esta recta tiene que tener pendiente $m=-\dfrac{1}{6}$.

$$ y=-\dfrac{1}{6}x+b $$

Puedo tomar cualquier $b$ y seguirá siendo paralela a la ecuación del ejemplo.

*(Básicamente, si queremos la pendiente perpendicular basta con que volteemos la fracción de nuestra pendiente y su signo.)*

* ***Ejemplo (de excepción):*** Tenemos la ecuación de la recta $y=0$ (recta horizontal). Construye otra recta que sea perpendicular.

$$ x=0\ \text{(recta vertical)} $$

* ***Ejemplo (de excepción):*** Tenemos la ecuación de la recta $y=b$ (recta horizontal) con constante. Construye otra recta que sea perpendicular.

$$ x=a\ \text{(recta vertical)} $$

### Ángulo entre dos rectas conociendo sus pendientes

Para encontrar el ángulo entre dos rectas

* primera recta, $y=m_1x+b_1$
* segunda recta, $y=m_2x+b_2$

utilizamos la fórmula

$$ \theta =\tan^{-1}\left(\dfrac{m_2-m_1}{1+m_2m_1}\right) $$

* ***Ejemplo*** Tenemos dos rectas:
    * La primera recta es $x+2y-1=0$
    * La segunda recta es $9x-3y+21=0$
    
    ¿Cuál es el ángulo que se forma entre mis dos rectas?

Primero pasamos las rectas a la forma ordenada-pendiente

$$x+2y-1=0$$

$$ 2y=1-x $$

$$ y=\dfrac{1-x}{2} $$

$$ y=\dfrac{1}{2}-\dfrac{x}{2} $$

$m_1=-\dfrac{1}{2}$, $b_1=\dfrac{1}{2}$

$$9x-3y+21=0$$

$$-3y=-9x-21$$

$$y=\dfrac{-9x-21}{-3}$$

$$y=\dfrac{9x}{3}+\dfrac{21}{3}$$

$$y=3x+7$$

$m_2=3$, $b_2=7$

$$ \theta =\tan^{-1}\left(\dfrac{m_2-m_1}{1+m_2m_1}\right) $$

$$ =\tan^{-1}\left(\dfrac{(3)-\left(-\dfrac{1}{2}\right)}{1+(3)\left(-\dfrac{1}{2}\right)}\right) $$

In [1]:
3-(-1/2)

3.5

In [2]:
1+(3)(-1/2)

-0.5

In [3]:
3.5/(-0.5)

-7.0

In [6]:
atand(-7)

-81.86989764584403

Entonces, el ángulo agudo entre las dos rectas es $81.87^\circ$

Entonces, el ángulo obtuso entre las dos rectas es $98.13^\circ$

## Distancia de un punto a una recta

Supongamos que tenemos una recta en su forma general
$$ Ax+By+C=0 $$

Pensemos que tenemos un punto $P(x_0,y_0)$

La distancia entre el punto y la recta es

$$ d=\dfrac{|Ax_0+By_0+C|}{\sqrt{A^2+B^2}} $$

**Reglas:**
1. Si me dan la ecuación de la recta en la forma ordenada-pendiente tengo que pasarla a su forma general. Si me la dan en la forma general vamos al siguiente paso
2. Señalamos cuánto valen los coeficientes de la recta en su forma general
3. Sustituimos las coordenadas del punto $P(x_0,y_0)$ en la forma general de la ecuación de la recta y evaluamos.

    Nota: Las líneas verticales en la fórmula significa *valor absoluto*, es decir, el valor que obtengamos lo mantemos positivo.

* ***Ejemplo:*** Tenemos la recta $y=x+2$ y los puntos $P(1,1)$ y $Q(2,4)$. 
    
    (a) ¿Cuál es la distancia entre la recta y el punto $P$?
    
    (b) ¿Cuál es la distancia entre la recta y el punto $Q$?

$$ y=x+2 $$

$$ -x+y-2=0 $$

$A=-1$, $B=1$, $C=-2$

Para la distancia respecto al punto $P$ tendremos

$$ d=\dfrac{|Ax_0+By_0+C|}{\sqrt{A^2+B^2}} $$

$$ =\dfrac{|(-1)(1)+(1)(1)+(-2)|}{\sqrt{(-1)^2+(1)^2}} $$

$$ =\dfrac{|-1+1-2|}{\sqrt{1+1}} $$

$$ =\dfrac{|-2|}{\sqrt{2}} $$

$$ =\dfrac{2}{\sqrt{2}}=\sqrt{2} $$

Para la distancia respecto al punto $Q$ tendremos

$$ d=\dfrac{|Ax_0+By_0+C|}{\sqrt{A^2+B^2}} $$

$$ =\dfrac{|(-1)(2)+(1)(4)+(-2)|}{\sqrt{(-1)^2+(1)^2}} $$

$$ =\dfrac{|-2+4-2|}{\sqrt{1+1}} $$

$$ =\dfrac{|0|}{\sqrt{2}}=0 $$

*"Cualquier punto que se encuentre sobre mi recta, tiene una distancia cero a esa recta."*

## El área de cualquier triángulo

### El área conociendo tres lados (Fórmula de Herón)

<img src="triangulo.png" align="middle" width="300">

$$ \text{Area}=\sqrt {s\left(s-a\right)\left(s-b\right)\left(s-c\right)}\ ,$$

donde $s=\dfrac{a+b+c}{2}=\dfrac{P}{2}$ es el semiperímetro (la mitad del perímetro).

## Rectas importantes en un triángulo

* *Mediana*: la recta que va del vértice al punto medio del cateto opuesto. El punto donde se unen las medianas se conoce como *baricentro*.

<img src="baricentro.png" align="middle" width="400">

* *Mediatriz*: la recta perpendicular al cateto y que pasa por el punto medio de éste. El punto donde se unen las mediatrices se conoce como *circuncentro*.

<img src="circuncentro.png" align="middle" width="400">

* *Altura*: la recta perpendicular que pasa por el vértice. El punto donde se unen las alturas se conoce como *ortocentro*.

<img src="ortocentro.png" align="middle" width="400">

* *Recta de Euler*: es una recta que, sin importar el triángulo del que se trate, pasa por el baricentro, circuncentro y ortocentro de dicho triángulo.

<img src="euler.png" align="middle" width="400">

**Propiedades:**
* En un triángulo equilátero el baricentro, el circuncentro y el ortocentro coinciden en el mismo punto.

Si tenemos un triángulo con vértices $A(x_1,y_1)$, $B(x_2,y_2)$, y $C(x_3,y_3)$, las coordenadas del

* baricentro son $(x_b,y_b)$, donde

\begin{align*}
x_b=\dfrac{x_1+x_2+x_3}{3}\\
y_b=\dfrac{y_1+y_2+y_3}{3}
\end{align*}

* circuncentro son $(x_c,y_c)$, donde

$$ x_c=\frac{x_1 \sin(2A)+ x_2 \sin(2B) + x_3 \sin(2C)}{\sin(2A)+\sin(2B)+ \sin (2C)} $$

$$ y_c=\frac{y_1 \sin(2A)+ y_2 \sin(2B) + y_3 \sin(2C)}{\sin(2A)+\sin(2B)+ \sin (2C)} $$

y $A$, $B$, y $C$ son los ángulos en los vértices.

* ortocentro son $(x_o,y_o)$, donde

\begin{align*}
x_o=\dfrac{[x_2(x_1-x_3)+y_2(y_1-y_3)](y_3-y_2)-(y_3-y_1)[x_1(x_2-x_3)+y_1(y_2-y_3)]}{(x_3-x_2)(y_3-y_1)-(y_3-y_2)(x_3-x_1)}\\
y_o=\dfrac{[x_2(x_1-x_3)+y_2(y_1-y_3)](x_3-x_2)-(x_3-x_1)[x_1(x_2-x_3)+y_1(y_2-y_3)]}{(y_3-y_2)(x_3-x_1)-(x_3-x_2)(y_3-y_1)}
\end{align*}

* ***Ejemplo:*** Consideremos un triángulo con vértices $A(-6,10)$, $B(8,6)$, y $C(8,-1)$.
  * Encuentra la ecuación de la recta que pasa por cada lado del triángulo.
  * Calcula el perímetro del triángulo.
  * Encuentra los ángulos internos del triángulo mediante ángulo entre pendientes.
  * Calcula el área del triángulo.
  * Encuentra la ecuación de todas las medianas del triángulo.
  * Obtén las coordenadas del baricentro (usando intersección de medianas).
  * Encuentra la ecuación de todas las mediatrices del triángulo.
  * Obtén las coordenadas del circuncentro (usando intersección de mediatrices).
  * Encuentra la ecuación de todas las alturas del triángulo.
  * Obtén las coordenadas del ortocentro (usando intersección de alturas).
  * Encuentra la ecuación de la recta de Euler.
  * Corrobora tus resultados con `Desmos`.

<img src="triangulo_euler.png" align="middle" width="500">

* Encuentra la ecuación de la recta que pasa por cada lado del triángulo.

 $A(-6,10)$, $B(8,6)$, y $C(8,-1)$

Ecuación de la recta que pasa por $A$ y $B$

$$ y=mx+b $$

$$ m=\dfrac{y_2-y_1}{x_2-x_1} $$

$$ =\dfrac{(6)-(10)}{(8)-(-6)}=\dfrac{-4}{14}=-\dfrac{2}{7} $$

$$ b=y-mx=(10)-\left(-\dfrac{2}{7}\right)(-6)=10-\dfrac{12}{7}=\dfrac{58}{7} $$

$$ y=-\dfrac{2}{7}x+\dfrac{58}{7} $$

https://www.desmos.com/calculator/ydpjmd53ik

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

https://www.desmos.com/calculator/wtivsqoik1

## Cónicas

Las cónicas son:
* circunferencia
* elipse
* parábola
* hipérbola

<img src="conicas.png" align="middle" width="700">

<img src="conic_sections.gif" align="middle" width="700">

<img src="conics_wolfram.gif" align="middle" width="500">

### Ecuación general de segundo grado

La ecuación general de las cónicas es:

$$ Ax^2+Bxy+Cy^2+Dx+Ey+F=0 $$

### Criterios para distinguir la cónica a partir de la ecuación de segundo grado

Cuando $B=0$ tendremos una:
* circunferencia si $A=C$
* parábola si $A=0$ ó $C=0$
* elipse si $A\neq C$ pero del mismo signo
* hiperbola si $A$ y $C$ tienen signos contrarios

$$ Ax^2+Cy^2+Dx+Ey+F=0 $$

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

* ***Ejemplo:*** ¿Cuál cónica representa la ecuación $2x^2+2y^2=7$?

$$  2x^2+2y^2-7=0 $$

$$ Ax^2+Bxy+Cy^2+Dx+Ey+F=0 $$

$A=$, $B=$, $C=$, $D=$, $E=$, $F=$

Como $A$ y $C$

* ***Ejemplo:*** ¿Cuál cónica representa la ecuación $y^2=1+x^2$?

$A=$, $B=$, $C=$, $D=$, $E=$, $F=$

* ***Ejemplo:*** ¿Cuál cónica representa la ecuación $x^2=8y$?

$$  x^2-8y=0 $$

$$ Ax^2+Bxy+Cy^2+Dx+Ey+F=0 $$

$A=$, $B=$, $C=$, $D=$, $E=$, $F=$

Como $C=$...

* ***Ejemplo:*** ¿Cuál cónica representa la ecuación $1=\dfrac{12-4y^2}{3x^2}$?

$ $

$$ Ax^2+Bxy+Cy^2+Dx+Ey+F=0 $$

$A=$, $B=$, $C=$

Como $A$ y $C$ ...

**¿Qué pasa cuando $B$ no es cero?**

Llamamos a $I=B^2-4AC$ el indicador.

Entonces, cuando $B\neq0$ ($B$ significa que la figura está rotada) tendremos una:
* parábola si $I=0$
* elipse si $I<0$
* hiperbola si $I>0$

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

* ***Ejemplo:*** ¿Cuál cónica representa la ecuación $2x^2-4xy+2y^2-40x+20y=0$?

$$ Ax^2+Bxy+Cy^2+Dx+Ey+F=0 $$

$A=$, $B=$ ,$C=$

$$ I= $$

Como el indicador es ______, se trata de una ______.

* ***Ejemplo:*** ¿Cuál cónica representa la ecuación $3x^2+2xy+3y^2-8y-2=0$?

$$ Ax^2+Bxy+Cy^2+Dx+Ey+F=0 $$

$A=$, $B=$, $C=$

$$ I= $$

Como el indicador es ______, se trata de una ______.